x-pack/plugins/session_view/public/components/process_tree_node/index.tsx

```diff
@@ -41,12 +43,13 @@ export function ProcessTreeNode({
   isSessionLeader = false,
   depth = 0,
   onProcessSelected,
+  timeStampOn,
+  verboseModeOn = true,
```

Comment:
might as well give `timeStampOn` a default value if there's one for `verboseModeOn`